crates/turborepo-lib/src/daemon/server.rs

```diff
@@ -604,7 +604,7 @@ impl proto::turbod_server::Turbod for TurboGrpcServiceInner {
             .package_changes()
             .await;
 
-        let (tx, rx) = mpsc::channel(1024);
+        let (tx, rx) = mpsc::channel(4096);
```

Comment:
I just wanted to surface the description for the commit-message as a motivation for this change, which should remove any element of surprise.

> Bumps buffers size for package change events from 1024 to 4096.
With some build tools (e.g. cargo) the number of changes to the build
direcotry can be quite large resulting in processing delay.
>
> It is also likely that during build operations the cpu resources are
constrained further contributing to delays in event processing,
thus increasing the buffer size seems appropriate